pepper-apis/studybuilder-cli/studies/basil/tabular-activity.conf

```diff
@@ -14,53 +14,66 @@
         {
           "headers": [
             {
-              "columnSpan":1,
+              "columnSpan": 2,
               "label": {"templateText": "$tabular.header.column_left"}
             },
             {
-              "columnSpan":1,
+              "columnSpan": 2,
               "label": {"templateText": "$tabular.header.column_right"}
             }
           ],
-          "rows": [
+          "questionBlocks": [
```

Comment:
If not major change, I am going to suggest we go with "blocks". Will allow us to be forwards compatible if we ever include other kinds of blocks.